google/cloud/_helpers.py

```diff
@@ -64,10 +66,10 @@
     _USER_ROOT = os.path.expanduser('~')
 except ImportError:  # pragma: NO COVER
     _USER_ROOT = None
-_GOOGLE_CLOUD_CONFIG_FILE = os.path.join(
+_GCLOUD_CONFIG_FILE = os.path.join(
     'gcloud', 'configurations', 'config_default')
-_GOOGLE_CLOUD_CONFIG_SECTION = 'core'
-_GOOGLE_CLOUD_CONFIG_KEY = 'project'
+_GCLOUD_CONFIG_SECTION = 'core'
+_GCLOUD_CONFIG_KEY = 'project'
```

Comment:
Yeah. The script was for env. vars. but these literally refer to the `gcloud` CLI